packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -124,7 +86,11 @@ function startedInDebugMode(): boolean {
   const args = (process as any).execArgv;
   if (args) {
     return args.some(
-      (arg: any) => /^--debug=?/.test(arg) || /^--debug-brk=?/.test(arg)
+      (arg: any) =>
+        /^--debug=?/.test(arg) ||
```

Comment:
This is needed since VS Code 1.15 and below use --debug-brk and 1.16 and above will use inspect--brk